simpleui/ViewController.swift

```diff
@@ -7,8 +7,112 @@
 //
 
 import UIKit
+import Darwin
+
 
 class ViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = UIColor(red: 254 / 255.0, green: 249 / 255.0, blue: 55 / 255.0, alpha: 1 / 1)
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    var t_stat:Int = 4
+    var t_statd:Double = 4
+    var kochzeit:Double = 6
+    var höhe:Int = 300
+    var höhed:Double = 300
+    var gewicht:Int = 50
+    var gewichtd:Double = 50
+    var kons:Int = 65
+    var konsd:Double = 65
+    
+    var druckd:Double = 0
+    var t_siedd:Double = 0
+    
+    
+    @IBOutlet var höhetext: UILabel!
+    
+    @IBOutlet var höheslider: UISlider!
+
+    @IBAction func höhechange(sender: AnyObject) {
+        höhe = Int(höheslider.value)
+        höhed = Double(höhe)
+        höhetext.text = "\(höhe)"
+        
+        druckd = (1013.25*exp(-höhed/8000))
```

Comment:
Den Berechnungscode schreibst du mehrmals, du solltest sowas in eine Funktion auslagern, z.B.:

```swift
func updateKochzeitLabel() {
    // read values from interface elements
    // calculate kochzeit
    // set kochzeit label text
}
```

Die Funktion kannst du dann einfach immer aufrufen, wenn sich etwas ändert:

```swift
updateKochzeitLabel()
```